src/core/persistence/persistence.service.d.ts

```diff
@@ -1,4 +1,6 @@
 export declare class PersistenceService {
 	save(settings?: obejct): object;
 	load(model: object, settings?: object): void;
+	active(model: object, settings?: object): boolean;
```

Comment:
isActive and stringify please put under persistence plugin, as they will be in core we would be able to reuse them